contracts/HubPool.sol

```diff
@@ -334,7 +335,7 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
      * @notice Sets root bundle proposal liveness period. Callable only by owner.
      * @param newLiveness New liveness period.
      */
-    function setLiveness(uint32 newLiveness) public override onlyOwner {
+    function setLiveness(uint32 newLiveness) public override onlyOwner nonReentrant {
```

Comment:
I personally feel that adding it to these admin functions is not needed at all. no real downside to having it to be as consistant as posible but it does not really do anything meaningful.

Reply:
Also see this comment I had added in the PR: https://github.com/across-protocol/contracts-v2/blob/cc6853a368e19075b0803c77cc0190fdf85e6216/contracts/SpokePool.sol#L27